### ISSUE TITLE
My 15-minute count sample page - ability to download route

### ISSUE THREAD

**Issue by DavidRoy** [BiologicalRecordsCentre/ABLE#363]
Add link for recorder to download the kml of the sample, e.g. from https://butterfly-monitoring.net/mydata/samples/details?sample_id=15342137

**Comment by DavidRoy** [comment]
@johnvanbreda is it possible to add the route download (kml) to this page for the specific sample?

**Comment by johnvanbreda** [comment]
@DavidRoy yes, we can add a sample ID filter to the existing reports used for KML download. Presumably this should include the visit and the point samples together? Also, is this only for a user viewing their own sample?
